docs/intentReaderHandlers/SetApiResponseHandler.md

```diff
@@ -0,0 +1,140 @@
+# SetApiResponseHandler 
+
+## Overview
+
+SetApiResponseHandler is invoked when the task specified in the intent has the value "setApiResponse". This handler is used to assign the apiResponse value to an environment variable in FCA.
+
+It performs the following actions:
+1. Parses the input message received to get the API response, which is to be assigned.
+2. Assigns the value to an environment variable.
+
+## Usage
+* This handler is invoked to assign the response value to an environment variable in FCA.
+
+### Request Format
+
+```json
+    {
+        "action": "search",
+        "data": {
+            "query": {
+                "task": "setApiResponse",
+                "params": {
+                    "apiResponse": {
+                        "module": "<moduleName>",
+                        "methodName": "<methodName>",
+                        "type": "method",
+                        "attributes": [
+                            {
+                                "..."
+                            }
+                        ]
+                    }
+                },
+                "action": "NA",
+                "appType": "<appType>"
+            }
+        },
+        "context": {
+            "source": "device"
+        }
+    }
+```
+#### Parameters
+
+| Key                   | Description                                                                              | Required? |
+|-----------------------|------------------------------------------------------------------------------------------|-----------|
+| task                  | "setApiResponse"- It is a static value and should not be changed for this handler          | Y         |
+| params                | Required response params for the intent. Here, "apiResponse" is a mandatory parameter    | Y         |
+| module                | Corresponding module name for which the API is called                                    | Y         |
+| methodName            | The name of the invoked method                                                           | Y         |
+| type                  | It indicates whether the response is of method/event                                     | Y         |
+| appType               | Corresponding intent is launching on which app                                           | Y         |
+
+### Response Format
+* Response can be either "true" or an error response
```

Comment:
So just want to make sure this is accurate because in the example valid response you have a "success message" instead of a "true". Please make sure the following things are all in sync with each other:
- `Response Format` description
- Response Parameters
- Example Responses

Please make sure these are aligned in all the handlers for both positive and negative scenarios, I think the majority are good already but have seen a few that looks out of place.